app/home/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+class Playlist(models.Model):
+    user_id = models.CharField(max_length=50)
```

Comment:
so for these I would do
```
owner = models.ForeignKey(User)
group = models.ForeignKey(Group)
name = models.CharField(max_length=100)
is_master = models.BooleanField(default=False)
spotify_playlist_id = models.CharField(max_length=255)  # we should confirm the format of these
```

Reply:
You could also ditch the `SongList` model and do `songs = models.ManyToManyField(Song)`